OracleSOASuite/dockerfiles/README.md

```diff
@@ -87,130 +21,49 @@ https://github.com/oracle/docker-images/tree/master/OracleJava/README.md
 
 Please refer [README.md](https://github.com/oracle/docker-images/blob/master/OracleFMWInfrastructure/README.md) under docker/OracleFMWInfrastructure for details on how to build Oracle FMW Infrastructure image.
 
-The Oracle FMW Infrastructure image created through above step need to be retagged from `container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3`  to `middleware/fmw-infrastructure:12.2.1.3` before continuing with next steps.
+The Oracle FMW Infrastructure image created through above step need to be retagged to the appropriate version of the image being created. e.g. for the 12.2.1.3.0 version, retag from `container-registry.oracle.com/middleware/fmw-infrastructure:12.2.1.3` to `oracle/fmw-infrastructure:12.2.1.3` before continuing with next steps.
```

Comment:
You don't need to retag a manually built image, as it'll be built with the required tag. You only need to retag if you pull the image from Oracle Container Registry.

Reply:
Fixed it.